src/components/MaterialUI/Inputs/Radio.js

```diff
@@ -116,12 +125,22 @@ const Radio = ({ radioProps }) => {
 									onClick={handleClick}
 								/>
 							}
-						/>
+						></FormControlLabel>
```

Comment:
Unless something is in the middle, can you just close the tag like it was before?

Reply:
Fixed!